build-in-container/run.sh

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash
```

Comment:
If we change this file to have a more descriptive name it can safely be added to a users $PATH etc and invoked quickly from anywhere